firebase-vscode/src/test/webpack.test.js

```diff
@@ -0,0 +1,56 @@
+const { merge } = require("webpack-merge");
+const path = require("path");
+const configs = require("../../webpack.common");
+const glob = require("glob");
+
+const extensionConfig = configs.find((config) => config.name === "extension");
+
+const getTestFiles = () =>
+  new Promise((resolve, reject) => {
+    glob(
+      "**/**.test.ts",
+      { cwd: path.resolve(__dirname, "suite") },
+      (err, files) => {
+        if (err) {
+          reject(e(err));
+        }
+        const testFiles = {};
```

Comment:
I'm guessing this is necessary? Didn't think we'd have to search for test files manually.

Reply:
We need to be very specific in telling webpack exactly which files are test suite files, and make sure not to include the test scaffolding files. On second thought I think maybe the webpack compile process is good enough to use on the test scaffolding files too and we don't need two compile processes. I think it's good to keep the processes separate somewhat as they do different things (runTest.ts/index.ts set up the test process, *.test.ts are the actual tests, which may import CLI code, dependencies, etc).

Or let me know if I misunderstood the question.